src/manager/manager.c

```diff
@@ -914,6 +914,7 @@ static int manager_name_owner_changed(sd_bus_message *m, void *userdata, UNUSED
 }
 
 bool manager_start(Manager *manager) {
+        hirte_log_infof("Starting hirte manager v%s", CONFIG_H_HIRTE_VERSION);
```

Comment:
Same as above, just `Starting hirte 0.5.0` should be enough

Reply:
sure